### PR TITLE
Added a section on ability to import/export schedules

### PR DESCRIPTION
**Pull request by suyogsainkar**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1730044

### CODE REVIEW

File: doc-General_Configuration/topics/Configuration.adoc [hunk 1]
```diff
@@ -2052,6 +2052,52 @@ To modify a schedule:
...
 . Click *Save*.
 
+
+[[import-export-schedule]]
+===== Importing and Exporting Schedules
+
+{product-title_short} provides the ability to import and export schedules using the `import_export_schedules.rb` script:
+
+----
+./tools/import_export_schedules.rb -h
```
Comment (sbulage) [inline doc-General_Configuration/topics/Configuration.adoc]: **Suggestion:** 
@suyogsainkar From which location need to execute it, can you please add exact path or create alias for it. ?

Comment (suyogsainkar) [reply doc-General_Configuration/topics/Configuration.adoc]: > **Suggestion:**
> @suyogsainkar From which location need to execute it, can you please add exact path or create alias for it. ?
@sbulage I haven't tested this myself. Can you please share info about the location/exact path to execute it? If not, I'll check with engineering. Thanks.

Comment (sbulage) [reply doc-General_Configuration/topics/Configuration.adoc]: Okay

Comment (sbulage) [reply doc-General_Configuration/topics/Configuration.adoc]: @suyogsainkar This is the full path where `import_export_schedules.rb` file resides.
```
cd /var/www/miq/vmdb/tools
import_export_schedules.rb -h
```

Comment (suyogsainkar) [reply doc-General_Configuration/topics/Configuration.adoc]: Thanks Satyajit!

**Comment by suyogsainkar** [comment]
@sbulage Thanks again for your review and feedback. I've applied your suggestion in the latest commit. Merging now.

**Pull request merged**